.circleci/config.yml

```diff
@@ -171,6 +171,8 @@ workflows:
               - PPS2
               - PPS3
               - PPS4
+              - PPS5
```

Comment:
Did this get removed?

Reply:
No, the opposite, the bucket number was increased.

I'm attempting to add an automated check, but it might benefit from someone with better bash/sed/awk foo